ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/virtual-source-nat.j2

```diff
@@ -1,5 +1,5 @@
 {# eos - virtual source-nat - introduced in 4.21 #}
-{% if virtual_source_nat_vrfs is defined and virtual_source_nat_vrfs is not none %}
+{% if virtual_source_nat_vrfs is arista.avd.defined %}
```

Comment:
This check is now included in natural_sort filter, so we no longer need to test before the for-loop, as long as it uses natural_sort.